### DISCUSSION TITLE
Dialog box at top of the page

### DISCUSSION THREAD

**Discussion by bilogic** [filamentphp/filament#9917] [Help] [answered]
![image](https://github.com/filamentphp/filament/assets/946010/ca1e2aad-444c-4ab6-9cb2-116c724ba941)

Is there built-in way to quickly reproduce the red box? (Otherwise I will have to build something myself)

**Comment by leandrocfe** [comment, marked as answer]
Maybe this?
https://filamentphp.com/plugins/awcodes-shout#usage-in-forms

**Reply by bilogic** [reply]
thanks let me try it out!
